### PR TITLE
Blog for IntelliJ 0.12.0

### PR DESCRIPTION
**Pull request by jopit**
Signed-off-by: John Pitman <jspitman@ca.ibm.com>

## What type of PR is this ? 

- [ ] Bug fix
- [x] Enhancement

## What does this PR do ?

Adds a blog entry for IntelliJ status for 0.12

### CODE REVIEW

**Comment by jopit** [comment]
@beccabau can you add a comment so I can add you as a reviewer?

**Comment by beccabau** [comment]
Yep!

**Comment by jopit** [comment]
@beccabau It's not letting me add you as a reviewer. Are you able to start a review yourself?

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+author: John Pitman
+blog_description: "Update on support for Codewind in IntelliJ IDEA"
+permalink: codewind-in-intellij-may
+duration: 2 minutes
+tags: [IntelliJ]
+---
+
+Over the past few months we've been hard at work creating a plugin for IntelliJ IDEA to add support for Codewind. This is a tech preview, because there are still some fundamental features to be implemented, but we have made significant progress.
+
+Our first tech-preview release included basic support for Codewind, and last month we added the ability to show the log files, open the performance dashboard, and debug projects.
```
Comment (beccabau) [inline docs/_blog/2020-05-14-codewind-in-intellij-may.md:16]: We have intro blog that you can link if you'd like ---
[first tech-preview release](https://www.eclipse.org/codewind/introducing-eclipse-codewind-on-intellij.html)

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+author_picture: images/blog/author_icon_john_pitman.png
+author_url:
+author: John Pitman
+blog_description: "Update on support for Codewind in IntelliJ IDEA"
+permalink: codewind-in-intellij-may
+duration: 2 minutes
+tags: [IntelliJ]
+---
+
+Over the past few months we've been hard at work creating a plugin for IntelliJ IDEA to add support for Codewind. This is a tech preview, because there are still some fundamental features to be implemented, but we have made significant progress.
```
Comment (beccabau) [inline docs/_blog/2020-05-14-codewind-in-intellij-may.md:14]: what about:
Over the past few months, we've been hard at work creating a Codewind plugin for IntelliJ IDEA.

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md (hunk 1 above)
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Changed

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md (hunk 2 above)
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Changed

**Comment by jopit** [comment]
@sishida the PR is looking for your approval, could you review/approve?

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+permalink: codewind-in-intellij-may
+duration: 2 minutes
+tags: [IntelliJ]
+---
+
+Over the past few months, we've been hard at work creating a Codewind plugin for IntelliJ IDEA. This is a tech preview, because there are still some fundamental features to be implemented, but we have made significant progress.
+
+Our [first tech-preview release](https://www.eclipse.org/codewind/introducing-eclipse-codewind-on-intellij.html){:target="_blank"} included basic support for Codewind, and last month we added the ability to show the log files, open the performance dashboard, and debug projects.
+
+This month, we've added the support for opening the application metrics dashboard, and the ability to open a shell into the application container.  Stay tuned for more function to be delivered next month.
```
Comment (sishida) [inline docs/_blog/2020-05-14-codewind-in-intellij-may.md:18]: - Remove comma after "dashboard"
- I'd prefer to remove the sentence, "Stay tuned...". I'm kinda wary about anything that sounds like a promise for the future.

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Here's a high-level feature comparison between the IDEs
```
Comment (sishida) [inline docs/_blog/2020-05-14-codewind-in-intellij-may.md:20]: Colon : after "IDEs"

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+| Feature |  VSCode  |  Eclipse	 |  IntelliJ  |
+| ------------- | ------------- | ------------- | ------------- |
+| Local container development | x | x | x | 
+|Automatic project build | x | x | x |
+|Remote codewind connection | x | x | |
```
Comment (sishida) [inline docs/_blog/2020-05-14-codewind-in-intellij-may.md:26]: Capitalize "Codewind" unless there's a particular reason not to.

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+Over the past few months, we've been hard at work creating a Codewind plugin for IntelliJ IDEA. This is a tech preview, because there are still some fundamental features to be implemented, but we have made significant progress.
+
+Our [first tech-preview release](https://www.eclipse.org/codewind/introducing-eclipse-codewind-on-intellij.html){:target="_blank"} included basic support for Codewind, and last month we added the ability to show the log files, open the performance dashboard, and debug projects.
+
+This month, we've added the support for opening the application metrics dashboard, and the ability to open a shell into the application container.  Stay tuned for more function to be delivered next month.
+
+Here's a high-level feature comparison between the IDEs
+
+| Feature |  VSCode  |  Eclipse	 |  IntelliJ  |
```
Comment (sishida) [inline docs/_blog/2020-05-14-codewind-in-intellij-may.md:22]: There is a space in "VS Code"

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+| ------------- | ------------- | ------------- | ------------- |
+| Local container development | x | x | x | 
+|Automatic project build | x | x | x |
+|Remote codewind connection | x | x | |
+|Launch Application | x | x | x |
```
Comment (sishida) [inline docs/_blog/2020-05-14-codewind-in-intellij-may.md:27]: I notice some of these features have both words capitalized and some only the first word. Is this a reflection of how the features appear in the UI? If so, then please leave them as-is. If not, we should just capitalize the first word and proper names and lower case other words. :)

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+author_picture: images/blog/author_icon_john_pitman.png
+author_url:
+author: John Pitman
+blog_description: "Update on support for Codewind in IntelliJ IDEA"
+permalink: codewind-in-intellij-may
+duration: 2 minutes
+tags: [IntelliJ]
+---
+
+Over the past few months, we've been hard at work creating a Codewind plugin for IntelliJ IDEA. This is a tech preview, because there are still some fundamental features to be implemented, but we have made significant progress.
```
[the remark this answers is not in the document]
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Fixed

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+Our [first tech-preview release](https://www.eclipse.org/codewind/introducing-eclipse-codewind-on-intellij.html){:target="_blank"} included basic support for Codewind, and last month we added the ability to show the log files, open the performance dashboard, and debug projects.
```
[the remark this answers is not in the document]
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Fixed

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md (hunk 3 above)
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Fixed

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md (hunk 4 above)
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Fixed

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md (hunk 5 above)
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Fixed

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md (hunk 6 above)
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Fixed

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md (hunk 7 above)
Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md]: Fixed

File: docs/_blog/2020-05-14-codewind-in-intellij-may.md [hunk 10]
```diff
@@ -11,30 +11,29 @@ duration: 2 minutes
...
-This month, we've added the support for opening the application metrics dashboard, and the ability to open a shell into the application container.  Stay tuned for more function to be delivered next month.
+This month, we've added the support for opening the application metrics dashboard and the ability to open a shell into the application container.
 
-Here's a high-level feature comparison between the IDEs
+Here's a high-level feature comparison between the IDEs:
 
-| Feature |  VSCode  |  Eclipse	 |  IntelliJ  |
+| Feature |  VS Code  |  Eclipse	 |  IntelliJ  |
 | ------------- | ------------- | ------------- | ------------- |
 | Local container development | x | x | x |
```
Comment (sishida) [inline docs/_blog/2020-05-14-codewind-in-intellij-may.md:24]: Will the space to the left of the text make the "Local container development" cell look out of alignment?

Comment (jopit) [reply docs/_blog/2020-05-14-codewind-in-intellij-may.md:24]: No, the generated table is aligned.

**Pull request merged**